core/state/statedb.go

```diff
@@ -1183,18 +1184,14 @@ func (s *StateDB) Commit(block uint64, deleteEmptyObjects bool) (common.Hash, er
 	// code didn't anticipate for.
 	workers.Go(func() error {
 		// Write the account trie changes, measuring the amount of wasted time
-		newroot, set, err := s.trie.Commit(true)
-		if err != nil {
-			return err
-		}
-		root = newroot
+		root, set = s.trie.Commit(true)
```

Comment:
i would prefer to define the `set` here. As it's only used inside of this function